meteor.js

```diff
@@ -1,244 +1,318 @@
+            }
+          }
         });
-    }
-}
+
+        print('[DEBUG] Files:', files, {primaryCSSfile});
+      }
+
+      // MAKE PATHS ABSOLUTE
+      if(_.isString(program.path)) {
+        // fix paths in the CSS file
+        if(!_.isEmpty(files.css)) {
+          _.each(files.css, (css, i) =>  {
+            var cssFile = fs.readFileSync(path.join(outputPath, css), { encoding: 'utf8' });
+            cssFile = cssFile.replace(/url\(\'\//g, `url('${program.path}`).replace(/url\(\//g, `url(${program.path}`);
+            fs.unlinkSync(path.join(outputPath, css));
+            fs.writeFileSync(path.join(outputPath, css), cssFile, { encoding: 'utf8' });
+
+            files.css[i] = `${program.path}${css}`;
+          });
+        }
+
+        if(!_.isEmpty(files.js)) {
+          _.each(files.js, (jsFile, i) =>  {
+            files.js[i] = `${program.path}${jsFile}`;
+          });
+        }
+
+        if (primaryCSSfile) {
+          primaryCSSfile = `${program.path}${primaryCSSfile}`;
+        }
+      } else {
+        if(!_.isEmpty(files.css)) {
+          _.each(files.css, (cssFile, i) =>  {
+            files.css[i] = `/${cssFile}`;
+          });
+        }
+
+        if(!_.isEmpty(files.js)) {
+          _.each(files.js, (jsFile, i) =>  {
+            files.js[i] = `/${jsFile}`;
+          });
+        }
+
+        if (primaryCSSfile) {
+          primaryCSSfile = `/${primaryCSSfile}`;
+        }
+      }
+
+      // ADD CSS
+      var css = [];
+      _.each(files.css, (cssFile) => {
+        css.push(`<link rel="stylesheet" type="text/css" href="${cssFile}">`);
+      });
+      css = css.join('');
+
+      if (RE.styles.template.test(content)) {
+        content = content.replace(RE.styles.template, css);
+      } else if (RE.styles.tag.test(content)) {
+        content = content.replace(RE.styles.tag, css);
+      }
+
+      if (RE.styles.url.test(content) && primaryCSSfile) {
+        content = content.replace(RE.styles.url, primaryCSSfile);
+      }
+
+      // ADD the SCRIPT files
+      var scripts = ['__meteor_runtime_config__'];
+      _.each(files.js, (jsFile) => {
+        scripts.push(`<script type="text/javascript" src="${jsFile}"></script>`);
+      });
+      scripts = scripts.join('');
+
+      // add the meteor runtime config
+      const settings = {
+        'meteorRelease': starJson.meteorRelease,
+        'ROOT_URL_PATH_PREFIX': process.env.ROOT_URL_PATH_PREFIX || '',
+        meteorEnv: {
+          NODE_ENV: 'production'
+        },
+        autoupdate: { versions: {}},
+        // 'DDP_DEFAULT_CONNECTION_URL': program.url || '', // will reload infinite if Meteor.disconnect is not called
+        // 'appId': process.env.APP_ID || null,
+        // 'autoupdateVersion': null, // "ecf7fcc2e3d4696ea099fdd287dfa56068a692ec"
+        // 'autoupdateVersionRefreshable': null, // "c5600e68d4f2f5b920340f777e3bfc4297127d6e"
+        // 'autoupdateVersionCordova': null
+      };
```

Comment:
Before this PR I had the issue, that `autoupdate` was not included in the `__meteor_runtime_config__` object and I had to manually reassign `autoupdate` to it or the client app would not correctly start. This is now not that case but unless you haven't explicitly made changes on the `__meteor_runtime_config__` (which I did not have spotted yet) then this may be a thing for a deeper test. Summarized: working but keep eyes on it.

Reply:
`{ autoupdate }` was added into `__meteor_runtime_config__`